### DISCUSSION TITLE
How do I add an item to the LazyVim greeter dashboard

### DISCUSSION THREAD

**Discussion by moonstruckmoth** [LazyVim/LazyVim#138] [Q&A] [answered]
I want to add a shortcut bitton to the existing shortcuts on the dashboard. 
I went through the example.lua, but extending alpha-vim seems different.

TIA

**Comment by folke** [comment, marked as answer]
```lua
  {
    "goolord/alpha-nvim",
    opts = function(_, dashboard)
      table.insert(
        dashboard.section.buttons.val,
        dashboard.button("f", " " .. " Find file", ":Telescope find_files <CR>")
      )
    end,
  },
```

**Reply by moonstruckmoth** [reply]
It worked. Thanks!

**Reply by lucasrabiec** [reply]
It worked but the colors don't apply. I have something like this:
```lua
local proj_button = dashboard.button("p", " " .. " Projects", ":Telescope projects<cr>")
proj_button.opts.hl = "AlphaButtons"
proj_button.opts.hl_shortcut = "AlphaShortcut"
table.insert(dashboard.section.buttons.val, 4, proj_button)
```
It works but I am wondering if there is shorter way to apply the colors? I tried put `{ hl = ..., hl_shorcut = ... }` as the last argument of `dashboard.button` but it doesn't work

**Reply by folke** [reply]
Check the alpha docs. This is not a LazyVim question

**Reply by lucasrabiec** [reply]
Yea I was just curious. I checked the source and it looks like there is no shorter way than this. `button` definition only allows set hl_shortcut in opts.

**Comment by Flimm** [comment]
Newer versions of LazyVim (v11.17.5) use `snacks.nvim` (not `alpha-nvim`), in particular, [`dashboard`](https://github.com/folke/snacks.nvim/blob/main/docs/dashboard.md).

To configure the greeter/dashboard, create a file named `~/.config/nvim/lua/plugins/snacks.lua` (change directory path as needed), with these contents:

```
return {
  "folke/snacks.nvim",
  optional = true,
  opts = {
    dashboard = {
      preset = {
        keys = {
          { icon = "🛈", key = "t", desc="Open NeovimTipsRandom", action=":NeovimTipsRandom" },
        },
      },
    },
  },
}
```

This will create a menu with one item which opens NeovimTipsRandom (assuming that plugin is installed).
